recipe/meta.yaml

```diff
@@ -1,46 +1,68 @@
 {% set name = "cufflinks" %}
 {% set version = "0.17.3" %}
-{% set build_number = 0 %}
 
 package:
-  name: python-{{ name|lower }}
+  name: python-{{ name|lower }}-suite
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 48c1b3406dc030004121966489eebc5518cea70fd4e3f16379b491328501a644
+  patches:
+    # `pandas.DataFrame.ix` is deprecated and does not work in newer versions.
+    - 83ef352c821f6f5092d4132e3ecd5d5eb6b4e9e8.patch
+    - fix_df_loc.patch
 
 build:
-  noarch: python
-  number: {{ build_number }}
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 0
 
 requirements:
+  build:
+    - m2-patch  # [win]
+    - patch     # [not win]
   host:
     - python
     - pip
-  run:
-    - python
-    - chart-studio >=1.0.0
-    - colorlover >=0.2.1
-    - ipython >=5.3.0
-    - ipywidgets >=7.0.0
-    - numpy >=1.9.2
-    - pandas >=0.19.2
-    - plotly >=4.1.1
-    - six >=1.9.0
-
-test:
-  imports:
-    - cufflinks
+    - setuptools
+    - wheel
 
 outputs:
   - name: python-cufflinks
+    build:
+      script: python -m pip install . --no-deps --no-build-isolation -vv
+    requirements:
+      build:
+        - m2-patch  # [win]
+        - patch     # [not win]
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python
+        - chart-studio >=1.0.0
+        - colorlover >=0.2.1
+        - ipython >=5.3.0
+        - ipywidgets >=7.0.0
+        - numpy >=1.9.2
+        # Removed Series.ix and DataFrame.ix in pandas 1.0.0, https://github.com/pandas-dev/pandas/pull/26438
+        - pandas >=1.0.0
```

Comment:
shouldn't this be `pandas>=0.19.2`?

Reply:
Because `DataFrame.ix` was deprecated and finally removed from `pandas 1.0.0` we use `pandas` **>=1.0.0**. We apply a patch to use `DataFrame.loc` instead of `DataFrame.ix`. That's why pandas **<1.0.0** will throw an error.